### DISCUSSION TITLE
Possibility to change the rootPath

### DISCUSSION THREAD

**Discussion by RobinBertilsson** [ivanhofer/typesafe-i18n#733] [Q&A]
Hi,

Is it possible to change the current "root path"? It seems to be `./src/i18n`, but in my setup i'd rather have it in the root directly, so just `./i18n`,

Thanks,
R

**Comment by ivanhofer** [comment]
Please take a look at the `outputPath` option: https://github.com/ivanhofer/typesafe-i18n/tree/main/packages/generator#outputpath
